src/components/ui/TripCardAccordion/TripCardAccordion.tsx

```diff
@@ -90,7 +91,8 @@ export const TripCardAccordion = ({
                         </Tooltip>
                         <Tooltip
                             title={
-                                data.seatType == ISeatType.SLEEPER
+                                data.seatType == ISeatType.SLEEPER ||
+                                data.seatType == ISeatType.Sleeper
```

Comment:
You can remove `Sleeper` from `ISeatType`. It will always be either `SLEEPER` or `SEATER`. The inconsistency is now handled in API layer.

Reply:
Resolved